src/main/java/se/bjurr/gitchangelog/internal/integrations/jira/JiraClient.java

```diff
@@ -25,30 +52,106 @@ public String getApi() {
     return this.api;
   }
 
+  protected String getIssuePath() {
+    return this.hasIssueFieldFilters() ? SEARCH_API : ISSUE_API;
```

Comment:
Can you not always use search API? When can you use it? When can you not use it?

Reply:
That made kinda sense to me, maybe?  I believe you would use `search` vs `issue` for when you are wanting an issue returned to _potentially_ be skipped. I tested this w/ Postman by adding essentially an `assignee = 'me@email.com'` and seeing the ticket either showing up or not.